packages/brisk-reconciler/brisk-reconciler.1.0.0~alpha1/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "A lightweight library for modeling tree-shaped state with stateful functions"
+description: """\
+Brisk Reconciler provides an expressive and powerful abstraction for managing **tree-shaped state** using **stateful functions**. This makes it ideal for modeling dynamic structures like the **DOM tree**, **app navigation state**, or even a **rich text document**.
+
+### ✨ **Key Features**
+- **Tree-shaped state modeling**: Represent and update hierarchical structures efficiently.
+- **OutputTree-agnostic hooks**: Support different output representations.
+- **UI-oriented, but flexible**: Designed with UIs in mind but applicable to other domains.
+"""
+maintainer: "brisk-reconciler developers"
+authors: ["Wojtek Czekalski"]
+license: "MIT"
+homepage: "https://github.com/briskml/brisk-reconciler"
+bug-reports: "https://github.com/briskml/brisk-reconciler/issues"
+depends: [
+  "dune" {>= "3.16"}
+  "ocaml" {>= "4.8.0" & < "5.3.0"}
+  "ppxlib"
+  "mlx" {with-test}
+  "ocamlformat-mlx" {with-dev-setup}
+  "ocamlformat" {with-dev-setup}
+  "ocaml-lsp-server" {with-dev-setup}
+  "alcotest" {with-test}
+  "ppx_deriving" {with-test}
+  "js_of_ocaml" {with-test}
+  "js_of_ocaml-ppx" {with-test}
+  "lwt" {with-test}
+  "lambda-term" {with-test}
+  "core_bench" {with-test & ocaml:version >= "5.1" & os != "win32"}
+  "core_unix" {with-test & ocaml:version >= "5.1" & os != "win32"}
```

Comment:
```suggestion
  "core_unix" {ocaml:version >= "5.1" & os != "win32"}
```
According to https://github.com/briskml/brisk-reconciler/blob/master/bench/dune the bench seems to be always compiled, so this should be an actual dependency

Reply:
I added a constraint `(enabled_if (= %{profile} dev))` to the bench executable dune file. The intent is that the library should not depend on `core_bench` and its dependencies unless installed with a dev profile. Does this satisfy the `core_unix` `with-test` constraint?

Reply:
I think it used to be the case that `with-test` would be on the `dev` profile, I am not sure if this has ever been changed. I'd leave it as is, since it seems to work and it should work during local development